test/IntegrationTests/TestAssets/DynamicDataTestProject/DynamicDataTests.cs

```diff
@@ -234,7 +234,7 @@ private static IEnumerable<object[]> GetExampleTestCases()
               }
             ]
             """;
-        return JsonConvert.DeserializeObject<ExampleTestCase[]>(json)
+        return JsonConvert.DeserializeObject<ExampleTestCase[]>(json)!
```

Comment:
Same here, it's redundant. I would avoid using `!` when it effectively does nothing (i.e, it doesn't suppress a warning)